rvc/train/mel_processing.py

```diff
@@ -76,19 +77,35 @@ def spectrogram_torch(y, n_fft, hop_size, win_size, center=False):
     )
     y = y.squeeze(1)
 
-    spec = torch.stft(
-        y,
-        n_fft,
-        hop_length=hop_size,
-        win_length=win_size,
-        window=hann_window[wnsize_dtype_device],
-        center=center,
-        pad_mode="reflect",
-        normalized=False,
-        onesided=True,
-        return_complex=True,
-    )
-
+    # Zluda fall-back to CPU for FFTs since HIP SDK has no cuFFT alternative
+    if y.device.type == "cuda" and torch.cuda.get_device_name(y.device.index).endswith("[ZLUDA]"):
```

Comment:
Same as in [rvc/lib/predictors/RMVPE.py](https://github.com/IAHispano/Applio/pull/513/files#diff-de128a8d02580f48332bab47d72e4d9af0892ec40cb2d7148d9ad05f55d820ee)